.editorconfig

```diff
@@ -0,0 +1,20 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+quote_type = single
+
+[{deps}/**]
+charset = unset
+end_of_line = unset
+indent_size = unset
+indent_style = unset
+trim_trailing_whitespace = unset
+
+[{test/fixtures,deps,tools/node_modules,tools/gyp,tools/icu,tools/msvs}/**]

```

Comment:
@vinibeloni Could you explain, why do we need to disable `insert_final_newline` for all this folders including `test` and `node_modules`?

Reply:
Oh sorry, I'll update this file and fix, based on the project structure